spec/dummy/client/app/startup/ClientApp.jsx

```diff
@@ -17,7 +17,7 @@ import HelloWorldContainer from '../components/HelloWorldContainer';
  *  React will see that the state is the same and not do anything.
  *
  */
-window.App = props => {
+let App = props => {
```

Comment:
We were trying to add a property to a function object. For some reason, we can't go 
```javascript
App.generator = true
```
The generator part is lost when the modules are bundled. Any ideas on why?